invoke-ai/InvokeAI
# Add Toggle for showing Bounding Box, fix Main Model info popover, fix Control Model description on hover

## Summary
small quality of life changes, including adding toggle for showing Bounding Box, fixing Main Model tooltip, and fixing Control Model description on hover


## Related Issues / Discussions
<img width="293" alt="Screenshot 2024-04-11 at 8 58 34 PM" src="https://github.com/invoke-ai/InvokeAI/assets/29263340/1dfcb79e-fef1-4d7b-b24e-7843e51c5b2a">

Closes #6195 



## QA Instructions
- Can hide and display bounding box using new button
- Can hover over the word "Model" for main model to see informational popover
- Can hover over the control model to see selected model's description


## Merge Plan



## Checklist
